### PR TITLE
📄Case Study - Template

### PR DESCRIPTION
**Pull request by amankumarrr**
- Affected routes: `/company/clients/shepherd-centre`
- Related #1328 

- [ ] Include done video or screenshots
shepherd-center

**Figure: Shepherd case study**

### CODE REVIEW

File: .tina/collections/case-study.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { tipField } from "./shared-fields";
+
+// export const caseStSchemaConstants = {
+//   value: "locations",
+//   header: "header",
+//   addressLine1: "addressLine1",
+//   addressLine2: "addressLine2",
+//   addressLine3: "addressLine3",
+//   directionURL: "directionURL",
+// };
```
Comment (bradystroud) [inline .tina/collections/case-study.tsx]: Do we need this comment?

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: .tina/collections/case-study.tsx (hunk 1 above)
Comment (amankumarrr) [reply .tina/collections/case-study.tsx]: I'm waiting for slot deployment, I'll do some clean up.

Comment (amankumarrr) [reply .tina/collections/case-study.tsx]: removed

**Review by bradystroud — CHANGES_REQUESTED** [review]
One of the breadcrumb links doesn't work

image

**Figure: Broken link**

File: public/images/POTY-Finalist-inclusion-logo.png
Comment (bradystroud) [inline public/images/POTY-Finalist-inclusion-logo.png:1]: I feel like we would already have this image somewhere.

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: public/images/POTY-Finalist-inclusion-logo.png
Comment (amankumarrr) [reply public/images/POTY-Finalist-inclusion-logo.png:1]: Yes, we are using it on `/company/news`

Comment (amankumarrr) [reply public/images/POTY-Finalist-inclusion-logo.png:1]: removed the one with this PR

**Comment by amankumarrr** [comment]
> One of the breadcrumb links doesn't work
> 
> image
> **Figure: Broken link**

As per my conversation with @wicksipedia, we don't need extra path (i.e `case-study`) for this. I've updated PBI and PR accordingly.

**Review by bradystroud — APPROVED** [review]
lgtm

**Pull request merged**